engine/src/main/java/org/terasology/engine/logic/location/LocationComponent.java

```diff
@@ -35,7 +36,7 @@
     // Standard position/rotation
     @Replicate
     @TextField
-    Vector3f position = new Vector3f();
+    final Vector3f position = new Vector3f();
```

Comment:
I don't think you can have this as final. how the the serialization work in gestalt, I thought it instantiated a new object and assigned it but you can't do that if the field is final.